doc/user_guide/install.rst

```diff
@@ -69,29 +69,29 @@ Installation is completed! To start using it, check the next section.
 3. Getting Started
 ------------------
 
-To get started using exSpy, especially if you are unfamiliar with Python, we
+To get started using eXSpy, especially if you are unfamiliar with Python, we
 recommend using `Jupyter notebooks <https://jupyter.org/>`_. Having installed
-exSpy as above, a Jupyter notebook can be installed and opened using the following commands
+eXSpy as above, a Jupyter notebook can be installed and opened using the following commands
 entered into an anaconda prompt (from scratch):
 
 .. code-block:: bash
 
-    (base) conda activate exspy
-    (exspy) conda install -c conda-forge jupyterlab -y
-    (exspy) jupyter lab
+    (base) conda activate eXSpy
+    (eXSpy) conda install -c conda-forge jupyterlab -y
+    (eXSpy) jupyter lab
 
 
 .. _pip:
 
 Installation using pip
 ======================
 
-Alternatively, you can also find exSpy in the `Python Package Index (PyPI) <https://pypi.org>`_
+Alternatively, you can also find eXSpy in the `Python Package Index (PyPI) <https://pypi.org>`_
 and install it using (requires ``pip``):
 
 .. code-block:: bash
 
-    pip install exspy
+    pip install eXSpy
```

Comment:
This may be working but usually library name are all lower case, wouldn't be better to use the lower case here too (and below)?

Reply:
Thought I did a case sensitive replace, but also all occurences in that file are fixed now, as well as some links.